src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public static void getRandomColor() {
+        Random random = new Random();
+        int index = new Random().nextInt(Color.values().length);

```

Comment:
You're creating two separate instances of Random. You should create only one instance and use it to generate the random index. Also, consider moving the Random instance to a class-level variable to avoid creating a new instance every time the method is called.